types/node/index.d.ts

```diff
@@ -6454,7 +6454,7 @@ declare module "http2" {
         writeContinue(): void;
         writeHead(statusCode: number, headers?: OutgoingHttpHeaders): void;
         writeHead(statusCode: number, statusMessage?: string, headers?: OutgoingHttpHeaders): void;
-        createPushResponse(headers: OutgoingHttpHeaders, callback?: (err: Error) => void): void;
+        createPushResponse(headers: OutgoingHttpHeaders, callback: (err: Error, res: Http2ServerResponse) => void): void;
```

Comment:
I think it should be `err: Error | null`. I know that such issues are at a lot other places so if you want to keep it as is it's ok.

Reply:
I thought of it when I made the first change and just tried to be consistent with other places. But I agree that `Error | null` is more correct. Made the change.